### PR TITLE
SSZ compatibility test for protobufs

### PR DESCRIPTION
**Pull request by terencechain**
This PR adds SSZ compatibility tests for protobufs

### CODE REVIEW

File: proto/testing/BUILD.bazel [hunk 1]
```diff
@@ -14,13 +14,33 @@ go_proto_library(
...
+    deps = [
+        "//proto/beacon/p2p/v1:go_default_library",
+        "//shared/testutil:go_default_library",
+        "@com_github_ghodss_yaml//:go_default_library",
+        "@com_github_prysmaticlabs_go_ssz//:go_default_library",
+    ],
+)
+
+proto_library(
+    name = "testing_proto",
```
Comment (prestonvanloon) [inline proto/testing/BUILD.bazel]: Why did you add this target? It's identical to ethereum_testing_proto

File: proto/testing/BUILD.bazel [hunk 2]
```diff
@@ -14,13 +14,33 @@ go_proto_library(
...
+    proto = ":testing_proto",
     visibility = ["//visibility:public"],
 )
 
 go_library(
     name = "go_default_library",
+    srcs = ["ssz_minimal.yaml.go"],
     embed = [":ethereum_testing_go_proto"],
     importpath = "github.com/prysmaticlabs/prysm/proto/testing",
     visibility = ["//visibility:public"],
```
Comment (prestonvanloon) [inline proto/testing/BUILD.bazel]:
```suggestion
    visibility = ["//visibility:public"],
    testonly = True,
```

File: proto/testing/BUILD.bazel (hunk 1 above)
Comment (terencechain) [reply proto/testing/BUILD.bazel]: The lint was complaining for `ethereal_testing_proto` (not suppose to have _), we can use `ethereumtesting` if you want

```
- package ethereum.testing;
+ package testing;
```

Comment (prestonvanloon) [reply proto/testing/BUILD.bazel]: OK, please remove the other target in that case.
